main.go

```diff
@@ -1,3 +1,51 @@
 package main
 
-func main() {}
+import (
+	"flag"
+	"fmt"
+	"log"
+	"net/http"
+	"os/exec"
+	"strings"
+)
+
+type logAggregator struct {
+	context    string
+	kubeconfig string
+	namespace  string
+}
+
+func (l logAggregator) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	var args []string
+	if l.namespace != "" {
+		args = append(args, []string{"-n", l.namespace}...)
+	}
+	if l.context != "" {
+		args = append(args, []string{"-c", l.context}...)
+	}
+	if l.kubeconfig != "" {
+		args = append(args, []string{"-k", l.kubeconfig}...)
+	}
+	runName := strings.TrimPrefix(r.URL.Path, "/")
+	args = append(args, []string{"pipelinerun", "logs", runName}...)
+
+	// #nosec
```

Comment:
Can you add a note about why this was necessary in here?

Reply:
Done

Reply:
I think the issue is the `Context()` call -- gosec might not complain if you call, store, and pass it instead, but I don't think it's a big deal to exclude this

Reply:
Actually it might still complain just for auditing -- just keep the `nosec` here

Reply:
It's probably not that important to use context here, but I'm trying to be more conscious of handling timeouts so processes don't get stuck.